ietf/submit/utils.py

```diff
@@ -671,20 +671,16 @@ def move_files_to_repository(submission):
             raise ValueError("Intended to move '%s' to '%s', but found source and destination missing.")
 
 
-def remove_staging_files(name, rev, exts=None):
-    """Remove staging files corresponding to a submission
-    
-    exts is a list of extensions to be removed. If None, defaults to settings.IDSUBMIT_FILE_TYPES.
-    """
-    if exts is None:
-        exts = [f'.{ext}' for ext in settings.IDSUBMIT_FILE_TYPES]
+def remove_staging_files(name, rev):
+    """Remove staging files corresponding to a submission"""
     basename = pathlib.Path(settings.IDSUBMIT_STAGING_PATH) / f'{name}-{rev}' 
+    exts = [f'.{ext}' for ext in settings.IDSUBMIT_FILE_TYPES]
```

Comment:
This is getting lucky and will only work as long as the set of generated filetypes is a subset of the submitable file types.

Reply:
Is there any reason not to just remove anything with the given stem?

Reply:
Not certain of the history, but I think `IDSUBMIT_FILE_TYPES` is essentially "extensions that might be associated with a submission" at this point. We use it in a couple places where we move files around. (See `rename_submission_files` and `move_files_to_repository`, at least.)

I'm inclined to keep it this way and fix it all when we refactor submission artifact handling.

Reply:
There'll be an issue to resolve when we're holding both xml2rfc and rfc2html ".html" files.